docs/lists.html

```diff
@@ -0,0 +1,137 @@
+                <li><a target="_blank" href="https://github.com/SinchanaMS/Clay" class="nav-subtitles">GitHub</a></li>
+            </ul>
+        </header>
+
+        <nav class="nav-bar">
+            <ul>
+                <h1 class="nav-title">Get Started</h1>
+                <li><a href="/docs/intro.html" class="nav-subtitles">Installation</a></li>
+            </ul>
+            <ul>
+                <h1 class="nav-title">Components</h1>
+                <li><a href="/docs/alerts.html" class="nav-subtitles">Alerts</a></li>
+                <li><a href="/docs/avatars.html" class="nav-subtitles">Avatars</a></li>
+                <li><a href="/docs/badges.html" class="nav-subtitles">Badges</a></li>
+                <li><a href="/docs/buttons.html" class="nav-subtitles">Buttons</a></li>
+                <li><a href="/docs/cards.html" class="nav-subtitles">Cards</a></li>
+                <li><a href="/docs/images.html" class="nav-subtitles">Images</a></li>
+                <li><a href="/docs/inputs.html" class="nav-subtitles">Inputs</a></li>
+                <li><a href="/docs/grids.html" class="nav-subtitles">Grids</a></li>
+                <li><a href="/docs/ratings.html" class="nav-subtitles">Ratings</a></li>
+                <li><a href="/docs/text-utilities.html" class="nav-subtitles">Text Utilities</a></li>
+                <li><a href="/docs/snackbars.html" class="nav-subtitles">Snackbars</a></li>
+                <li class="active-page"><a href="/docs/lists.html" class="nav-subtitles">Lists</a></li>
+            </ul>
+        </nav>
+
+        <section class="content">
+            <div class="component-wrapper">
+                <h1 class="wrapper-heading">Lists</h1>
+                <hr>
+                <div class="description">
+                    Lists are continuous, vertical indexes of text or images. We have Spaced Lists and Stacked Lists
+                    below.
+                </div>
+            </div>
+
+            <div class="component-wrapper lists-spaced-wrapper">
+                <h1 class="wrapper-heading">Spaced Lists</h1>
+                <hr>
+                <div class="description">
+                    These are list items aligned horizontally with consistent spacing.
+                    <br>
+                    Use the class <code class="class-highlight">spaced-lists</code> in the <code
+                        class="class-highlight">&ltul&gt</code> tag.
+                    <br><br>
+                    Here are some spaced list items:
+                </div>
+
+                <div class="component-showcase lists-spaced">
+                    <ul class="spaced-lists">
+                        <li>Item 1</li>
+                        <li>Item 2</li>
+                        <li>Item 3</li>
+                    </ul>
+                </div>
+
+                <pre><code class="code-snippet">
+<!-- add class="spaced-lists" in the <ul> tag-->
```

Comment:
remove the comment if no longer useful

Reply:
This comment is within the code snippet, put intentionally.